data-model/annotations/schema/annotation.json

```diff
@@ -2,99 +2,207 @@
   "$schema": "http://json-schema.org/draft-2020-12/schema#",
   "type": "object",
   "properties": {
-    "ods:id": { "type": "string" },
-    "ods:type": { "const": "Annotation" },
-    "version": { "type": "integer" },
+    "ods:id": {
+      "type": "string"
+    },
+    "ods:type": {
+      "const": "Annotation"
+    },
+    "version": {
+      "type": "integer"
+    },
```

Comment:
I think @samleeflang mentioned during the planning that version can have 'ods' prefix